securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -1,395 +1,360 @@
-As long as the same configuration file is used for all client operations, the developer will not need to
-authenticate in order to read, update, or delete a client that was created this way.
+The Client Registration CLI automatically uses its private configuration file to save and use this token with its associated client. As long as the same configuration file is used for all client operations, the developer does not need to authenticate to read, update, or delete a client that was created this way.
 
+See <<_client_registration, Client Registration>> for more information about Initial Access and Registration Access Tokens.
 
-You can read more about Initial Access and Registration Access Tokens in <<_client_registration,Client Registration chapter>>.
-
-See `kcreg config initial-token --help` and `kcreg config registration-token --help` for more information on how to configure them with `Client Registration CLI`.
-
+Run the [command]`kcreg config initial-token --help` and [command]`kcreg config registration-token --help` commands for more information on how to configure tokens with the Client Registration CLI.
 
 
 [[_performing_crud_operations]]
-==== Creating client configuration
+==== Creating a client configuration
 
-After authenticating with credentials or configuring Initial Access Token, the first operation will usually be to create a new client.
+The first task after authenticating with credentials or configuring an Initial Access Token is usually to create a new client. Often you might want to use a prepared JSON file as a template and set or override some of the attributes.
 
-We've seen the simplest create command already. Often we may want to use a prepared JSON file as a template and set / override
-some of the attributes. For example, here is how you read a JSON file, override any `clientId` it may contain,
-set any other attributes as well, and after successful creation print the configuration to standard output.
+The following example shows how to read a JSON file, override any client ID it may contain, set any other attributes, and print the configuration to a standard output after successful creation.
 
-On Linux:
-[source,bash]
+* Linux:
+[options="nowrap"]
 ----
 $ kcreg.sh create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s 'redirectUris=["/myclient/*"]' -o
 ----
-
-On Windows:
-[source,bash]
+* Windows:
+[options="nowrap"]
 ----
 C:\> kcreg create -f client-template.json -s clientId=myclient -s baseUrl=/myclient -s "redirectUris=[\"/myclient/*\"]" -o
 ----
 
+Run the [command]`kcreg create --help` for more information about the [command]`kcreg create` command.
 
-See `kcreg create --help` for more information about `kcreg create`.
+You can use [command]`kcreg attrs` to list available attributes. Keep in mind that many configuration attributes are not checked for validity or consistency. It is up to you to specify proper values. Remember that you should not have any ID fields in your
```

Comment:
In this sentence 'ID' is semantically invalid. It has to be 'id' because it refers to actual attribute name inside a JSON file. And yes, in this case it actually refers to 'id' and not 'clientId' :)